packages/ember-data/tests/integration/store-test.js

```diff
@@ -1,3 +1,5 @@
+import { badIdFormatAssertion } from 'ember-data/system/store';
```

Comment:
This causes an error:

~~~
 706 tests complete (9256 ms)
  706) integration/snapshot - DS.Snapshot: global failure
     Error: Could not find module ember-data/system/store
~~~

Is there any other way I can import exported module variables? I wonder why I can't use normal import like in HTMLBars. I also haven't found any example of using `import` in unit tests. I doubt they are real unit tests if they have access to all global namespaces, no imports. Is this because of old Ember CLI version?

PS: Tests on my local machine pass, appveyor build runs fine, only travis errors.

@wecc @bmac